libs/openFrameworks/graphics/ofPath.h

```diff
@@ -39,6 +39,8 @@ class ofPath{
 	/// \brief Create a new ofPath instance.
 	ofPath();
 
+	ofPath(ofPolyline &polyline);
```

Comment:
I think `const ofPolyline& polyline` would make more sense because we don't want to imply that the `ofPath` might change the polyline passed by reference.

Reply:
Additionally, please add documentation of the new constructor a e.g.

```c++
/// \brief Construct a new ofPath from the given ofPolyline.
///
/// A single polyline will be constructed from the given polyline and the ofPath mode will be set to `POLYLINES`.
///
/// \param polyline The polyline to that defines the new ofPath.

Reply:
will do!